roles/original_refspec_set_fact/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# vim: set ft=ansible:
+#
+# This role configures some facts based on the presence of an INI file that
+# holds some key=value pairs related to the original deployment on a host
+# under test.  This role is designed to be used after the role
+# `booted_deployment_set_fact` has been run.
+#
+# (I will probably regret making such a specific role...)
+#
```

Comment:
Can we template the variables as a yaml file and pull it in with include_vars?  Then we can have a generic role that pulls in vars from a specified file.  TBH, I haven't worked much with templating so I don't know if this will work.